CMakeLists.txt

```diff
@@ -59,10 +59,11 @@ option(WHISPER_BUILD_EXAMPLES         "whisper: build examples" ${WHISPER_STANDA
 
 option(WHISPER_SDL2                   "whisper: support for libSDL2" OFF)
 
-option(WHISPER_NO_AVX                 "whisper: disable AVX"  OFF)
-option(WHISPER_NO_AVX2                "whisper: disable AVX2" OFF)
-option(WHISPER_NO_FMA                 "whisper: disable FMA"  OFF)
-option(WHISPER_NO_F16C                "whisper: disable F16c" OFF)
+option(WHISPER_NO_AVX                 "whisper: disable AVX"    OFF)
+option(WHISPER_NO_AVX2                "whisper: disable AVX2"   OFF)
+option(WHISPER_AVX512                 "whisper: enable AVX512"  OFF)
```

Comment:
I suggest using `WHISPER_NO_AVX512` to be consistent with rest of these settings (until they will be overhauled one day in whisper.cpp to use positive notion instead of negative one), and naturally changing default to `ON` in such case.

Sorry if my rant in side note made you misunderstand the expectation for this PR.

Reply:
Are you suggesting to use `WHISPER_NO_AVX512` set to `ON` by default?

Reply:
Yes, to be consistent, because all other ISA extension options in whisper.cpp have such polarity.
<sub>(Maybe we'll change it in the future, but if it will happen, then it should be done separately.)</sub>